jme3-core/src/test/java/com/jme3/math/QuatMultTest.java

```diff
@@ -0,0 +1,133 @@
+
+//        final int reps = 100000;
+        final int reps = 10;
+         
+        for(int i = 0; i < reps; i++) {
+            randQuat(quat);
+            
+            randVect(vect);
+            
+            quat.mult(vect, store60);
+            mult15V2(quat, vect, store15);
+    
+            //eps stands for the epsillon in the approximateEquals method
+            //set your desired eps
+            
+            //final float eps = 0.00001f; 
+            //final float eps = 0.001f; 
+            final float eps = 0.08f;
+            
+            final boolean equals = approximateEquals(store60, store15, eps);
+            if(!equals) {
+                //print data if they are not equal with given epsillon
+                System.out.println("Iteration: " + i);
+                System.out.println("quat " + quat + ", vect " + vect );
+                System.out.println("store60 " + store60 + ", store15 " + store15);
+                System.out.println("diff " + store60.subtract(store15));
+                
+                System.out.println("("+approximateEquals(store60.x, store15.x, eps)+","+
+                        approximateEquals(store60.y, store15.y, eps)+","+
+                        approximateEquals(store60.z, store15.z, eps)+")");
+                //Failing this test means that the two methods produce results that 
+                //are different with respect to the eps
+                
+                //To determine which method provides more accurate result, use
+                //your favourite maths software or calculator to calculate the
+                //quaternion vector product from the data provided and compare
+                //the results
+            }
+            assertTrue(equals);
+        }        
+    }
+    public static boolean approximateEquals(Vector3f a, Vector3f b, float eps) {
+        return approximateEquals(a.x, b.x, eps) && approximateEquals(a.y, b.y, eps) &&
+                approximateEquals(a.z, b.z, eps);
+    }
+    public static boolean approximateEquals(float a, float b, float eps) {
+        if (a == b) {
+            return true;
+        } else {
+            return (abs(a - b) / Math.max(abs(a), abs(b))) <= eps;
+        }
+    }
+    
+    public void randQuat(Quaternion quat) {
+        quat.fromAngles(TWO_PI*rand.nextFloat(), TWO_PI*rand.nextFloat(), TWO_PI*rand.nextFloat());
+    }
+    public void randVect(Vector3f v) {
+        v.set(rand.nextFloat(), rand.nextFloat(), rand.nextFloat());
+        v.multLocal(10000f).subtractLocal(5000f, 5000f, 5000f);
+    }
+    public static void mult15V2(Quaternion q, Vector3f v, Vector3f s) {
+        //15 mult, 15 add
+        float x = q.getX();
+        float y = q.getY();
+        float z = q.getZ();
+        float w = q.getW();
+        
+        //v + 2*q.xyz cross (q.xyz cross v + w*v )
+        //q.xyz x v.xyz + w*v
+        float vx = y*v.z - z*v.y + w*v.x;
```

Comment:
Yes, sorry for this. First I made a test and uploaded it to pastebin. Then I forgot I have to change this since the new build will use the new method. I'll fix it now.